data/src/main/java/com/example/data/service/utils/extension/ImageResponseGetUrl.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.data.service.utils.extension
+
+import com.example.data.service.response.ImageResponse
+import com.example.data.service.utils.Constants.DOT
+
+fun ImageResponse.getURL(): String {
```

Comment:
it is correct that this method is in the data layer? as you are using this on app layer, this should be there, right?

Reply:
I put this extension function in the data layer because I use the method "GetURL" on another extension function (DataResponseExtension)